AsyncDisplayKit/ASImageNode.mm

```diff
@@ -200,6 +200,11 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 
 - (void)setImage:(UIImage *)image
 {
+  [self __setImage:image];
+}
+
+- (void)__setImage:(UIImage *)image
```

Comment:
Can we add a comment: "This is exposed to library subclasses, i.e. ASNetworkImageNode, ASMultiplexImageNode and ASVideoNode"